source/opt/constants.h

```diff
@@ -120,6 +120,27 @@ class IntConstant : public ScalarConstant {
   IntConstant* AsIntConstant() override { return this; }
   const IntConstant* AsIntConstant() const override { return this; }
 
+  int32_t GetS32BitValue() const {
+    assert(words().size() == 1);
```

Comment:
This relies critically on the fact that SPIR-V requires signed integer literal values be sign-extended.  It's worth saying that. (Section 2.2.1, definition of Literal Number.)

Reply:
Done.